### ISSUE TITLE
How to send infinite loop sequence?

### ISSUE THREAD

**Issue by anthonymich01** [cajhin/capsicain#103]
I have this sequence for press skill hotkey in MMO game

COMBO F [] > sequence(Q_W_E_A_S_D)

When I tap "F" it works, but I want the sequence not end in only 1 cycle, so it keeps press "Q_W_E_A_S_D" over and over again until I tap "F" again, is there a way to achieve this result? thank you in advance!

**Comment by cajhin** [comment]
No, you can't have an endless loop, this would require fundamental changes to the code. I'll keep this open as an enhancement, in case I have an idea how to solve this, but I don't think this will work.
